pyqt-apps/siriushla/as_ap_pvsconfmgr/read_configuration.py

```diff
@@ -221,11 +208,11 @@ def _is_configuration_valid(self):
 if __name__ == '__main__':
     import sys
     from siriushla.sirius_application import SiriusApplication
-    from siriuspy.servconf.conf_service import ConfigService
+    from siriuspy.clientconfigdb import ConfigDBClient
 
     app = SiriusApplication()
-    db = ConfigService('http://10.0.7.55:8085')
-    w = ReadConfigurationWindow(db)
+    client = ConfigDBClient(url='http://10.0.7.55:8085')
```

Comment:
Isn't it better to use `envars.server_url_configdb` instead of `'http://10.0.7.55:8085'`?

Reply:
thank you @anacso17! I've already fixed this in my last commit.